discoveries/no2-and-so2.discoveries.mdx

```diff
@@ -111,7 +111,7 @@ thematics:
       layerId='OMSO2PCA-COG'
       dateTime='2005-01-01'
       zoom={4}
-      center={[77.63,24.27]}
+      center={[14.5498259,-90.746572]}
```

Comment:
Just want to make sure if this change was intended?

Reply:
oops. i must have edited the wrong .mdx. I changed it back.